core/build.gradle

```diff
@@ -3,6 +3,14 @@ plugins {
     id 'maven-publish'
     id 'antlr'
     id "org.sonarqube"
+    id "com.diffplug.spotless" version "6.7.2"
+}
+
+spotless {
+    java {
+        target 'src/*/java/**/*.java'
+        removeUnusedImports()
+    }
 }
```

Comment:
This is only affecting to java files, but I believe there's some Kotlin in the database as well right?